drafts/inlinemacros.md

```diff
@@ -165,99 +170,66 @@ positions:
 Inline types can specifically not be used as
 
  - types of abstract methods or fields
- - types of variables
- - types of class parameters
+ - types of vars
+ - types of parameters to non-inline classes
  - type parameters to methods
  - type parameters in `new` expressions or class parents.
 
-## Inline Overloading Resolution
```

Comment:
I might be mistaken, but isn't this section obsoleted by the recently introduced by-value/by-name thingie?